.github/workflows/release.yml

```diff
@@ -95,3 +103,68 @@ jobs:
 
       - name: Smoke Test Tarball Files
         run: ./release/smoke-tests/run-tarball-files-smoke-tests.sh -v ${{ env.version }} -u ${{ secrets.ARCHIVES_PUBLIC_URL }} -n ${{ github.run_number }} -i ${{ matrix.image }} -t ${{ matrix.archive }}
+
+  protomote:
+    runs-on: ubuntu-latest
+    if: success() || failure()
+    needs: [build, validate-docker, validate-archive]
+    permissions:
+      contents: write
+      issues: write
+
+    steps:
+      - name: Checkout Data Prepper
+        uses: actions/checkout@v3
+      - name: Get Version
+        run: grep '^version=' gradle.properties >> $GITHUB_ENV
+
+      - name: Get Approvers
+        id: get_approvers
+        run: |
+          echo "approvers=$(cat .github/CODEOWNERS | grep @ | tr -d '* ' | sed 's/@/,/g' | sed 's/,//1')" >> $GITHUB_OUTPUT
+      - uses: trstringer/manual-approval@v1
+        with:
+          secret: ${{ github.TOKEN }}
+          approvers: ${{ steps.get_approvers.outputs.approvers }}
+          minimum-approvals: 2
+          issue-title: 'Release Data Prepper : ${{ env.version }}'
+          issue-body: >
+            Please approve or deny the release of Data Prepper.
+
+            **VERSION**: ${{ env.version }}
+
+            **BUILD NUMBER**: ${{ github.run_number }}
+
+            **RELEASE MAJOR TAG**: ${{ github.event.inputs.release-major-tag }}
+
+            **RELEASE LATEST TAG**: ${{ github.event.inputs.release-latest-tag }}
+
+          exclude-workflow-initiator-as-approver: false
+
+      - name: Create Release Description
```

Comment:
Steps from here will only be executed after getting 2 approvals right?

Reply:
Yes, this will require 2 approvals. The workflow initiator may be one of those approvers as well.